### DISCUSSION TITLE
Stuck at LineageOS Boot Screen after flashing from the Recovery

### DISCUSSION THREAD

**Discussion by ZeroHart** [pgale/lineage_15.1_Installation_TSV#25] [Q&A]
I have tried both in Windows and Linux ( I am terrible in Linux so I went back to Windows)
I was able to flash the recovery and everything and sideloaded the LineageOS image as well as Gapps. With renabling ADB between each sideload. 

The Problem is that when rebooting it just gets stuck in the boot screen for hours. 
Also when I checked both the windows and LInux instructions In windows it says to Factory reset then sideload. where in linux says to sideload and then factory reset. Tried both ways with no avail.

**Comment by Nathaaniel** [comment]
Did you manage to fix this? I'm having the same issue

**Reply by ZeroHart** [reply]
Yes, basically repeated every single step all over again from scratch and it worked
